### PR TITLE
Pull Request for Issue2429: REIXS sample camera crosshairs manual positioning

### PR DESCRIPTION
**Pull request by epengr**
A really quick change to give requested functionality.

### CODE REVIEW

**Comment by iainworkman** [comment]
Looks good to me.
Time Estimate: 1 minute.

File: source/ui/AMBeamlineCameraWidget.cpp [hunk 1]
```diff
@@ -141,7 +142,12 @@ void AMBeamlineCameraWidget::onVideoWidgetDoubleClicked(const QPointF &clickPoin
 void AMBeamlineCameraWidget::onRecenterCrosshairPushbuttonClicked()
 {
 	if(!crosshairLocked_)
-		setCrosshairPosition(crosshairCenter_);
+	{
+
+		double x = QInputDialog::getDouble(this, "Crosshair Position", "X coordinate(0-1):", crosshairCenter_.x(),0,1,4);
```
Comment (dretrex) [inline source/ui/AMBeamlineCameraWidget.cpp]: Does this pop up two independent, likely overlapping, dialog boxes?

**Comment by dretrex** [comment]
One usability question.
Time Estimate: 1 minute

File: source/ui/AMBeamlineCameraWidget.cpp (hunk 1 above)
Comment (epengr) [inline source/ui/AMBeamlineCameraWidget.cpp]: They are modal and therefore presented sequentially.  It's super user friendly, actually.  click recenter - type x value - enter - type y value - enter.  done.

Comment (dretrex) [inline source/ui/AMBeamlineCameraWidget.cpp]: Alright.  Sounds good.

**Pull request merged**